src/main/java/me/svdev/mscategory/dao/repository/CategoryRepository.java

```diff
@@ -0,0 +1,7 @@
+package me.svdev.mscategory.dao.repository;
+
+import me.svdev.mscategory.dao.entity.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
```

Comment:
[Optional] - You don't need everything from JpaRepository you can use also CrudRepository